### PR TITLE
Updated elsys so it sends also configuration topic

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/mclimate/vicki/uplink.js [hunk 1]
```diff
@@ -271,13 +271,13 @@ function consume(event) {
...
   const lifecycle = {};
-  let raw = {};
+  const raw = {};
 
   if (Number(bytes[0]) === 1 || Number(bytes[0]) === 129) {
     data = handleKeepalive(bytes, data);
   } else {
     data = handleResponse(bytes, data);
-    bytes = bytes.slice(-9);
+    bytes = Array.from(bytes).slice(-9);
```
Comment (mariusgiger) [inline types/mclimate/vicki/uplink.js:280]: make sure to properly test this change before moving to production

File: types/elsys/ers/uplink.js [hunk 2]
```diff
@@ -191,71 +408,83 @@ function deleteUnusedKeys(data) {
...
 function consume(event) {
-  const res = DecodeElsysPayload(Hex.hexToBytes(event.data.payloadHex));
-  const data = {};
-  const lifecycle = {};
-  const occupancy = {};
-  const noise = {};
+  const payload = Hex.hexToBytes(event.data.payloadHex);
+  const { port } = event.data;
+
+  if (port === 5) {
```
Comment (mariusgiger) [inline types/elsys/ers/uplink.js:414]: is it always port 5?

File: types/elsys/ers/configuration.schema.json [hunk 3]
```diff
@@ -0,0 +1,426 @@
+{
+  "$id": "https://akenza.io/elsys/ers/configuration.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Configuration",
+  "type": "object",
+  "properties": {
+    "appSKey": {
```
Comment (mariusgiger) [inline types/elsys/ers/configuration.schema.json:7]: outputting keys as data does not seem to be best practice, but if it is the Elsys way we should support it

**Review by mariusgiger — COMMENTED** [review]
looks good, please make sure (100%) that measurements are always port 5 otherwise change the logic to always fallback to parsing it is measurement

if(port ==6) {//parse settings} else { //parse measurement }

File: types/mclimate/vicki/uplink.js (hunk 1 above)
Comment (J2thatsme) [reply types/mclimate/vicki/uplink.js:280]: This works already testet in EU1

File: types/elsys/ers/configuration.schema.json (hunk 3 above)
Comment (J2thatsme) [reply types/elsys/ers/configuration.schema.json:7]: Usually they are not sent, but they could.

**Review by sevyakenza — APPROVED** [review]

**Pull request merged**